<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightSettingsUI</class>
 <widget class="QWidget" name="RightSettingsUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>521</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="tbRight">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
/*
QScrollBar::vertical {
	 width: 10px; 
}
*/




.QToolBox
{
 
    border-left: 1px solid rgb(0,106,255);
    border-bottom: 1px solid rgb(0,106,255);
    border-right: 1px solid rgb(0,106,255);
    border-radius: 3px;
}


QToolBox::tab{
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
	 font: bold 12pt Calibri;
	 color : rgb(127,180,255);
   	border-bottom: 1px solid rgb(0,106,255);
   	border-top: 1px solid rgb(0,106,255);
}


QToolBox::tab:hover{
	color : rgb(0,106,255);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}


QToolBox::tab:selected {
    color : rgb(0,106,255);
    background-color:  rgb(220,220,220);
	qproperty-alignment: AlignCenter;
    font: italic bold 14pt Calibri;
	padding-left: 0px;	

}



QToolBox::tab:!enabled{	
    background-color:  rgba(200,200,200,0%);
  /*  border: 1px solid rgba(200,200,200,0%);*/
 color : rgb(127,180,255);
}

QToolBox::tab:first{
    border-top: 1px solid rgb(0,106,255);
}


QToolBox::tab:!selected:last{
    border-bottom: 0px solid rgb(0,106,255);
}

QToolBox::tab:selected:last{
    border-bottom: 1px solid rgb(0,106,255);
}



</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabSpacing">
      <number>6</number>
     </property>
     <widget class="QWidget" name="pageAllPages">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/pages/nb_pages</normaloff>:/images/pages/nb_pages</iconset>
      </attribute>
      <attribute name="label">
       <string>POUR TOUTES LES PAGES</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlAllPages">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSelectedPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/pages/page</normaloff>:/images/pages/page</iconset>
      </attribute>
      <attribute name="label">
       <string>PAGE N°...</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlSelectedPage">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAllSets">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/sets/nb_sets</normaloff>:/images/sets/nb_sets</iconset>
      </attribute>
      <attribute name="label">
       <string>POUR TOUS LES ENSEMBLES (Photo+Texte)</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlAllSets">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSelectedSet">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/sets/set</normaloff>:/images/sets/set</iconset>
      </attribute>
      <attribute name="label">
       <string>ENSEMBLE (Photo+Texte) N°...</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlSelectedSet">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pageHeader">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/pages/header</normaloff>:/images/pages/header</iconset>
      </attribute>
      <attribute name="label">
       <string>HAUT DE PAGE</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlHeader">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFooter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>340</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/pages/footer</normaloff>:/images/pages/footer</iconset>
      </attribute>
      <attribute name="label">
       <string>BAS DE PAGE</string>
      </attribute>
      <layout class="QVBoxLayout" name="vlFooter">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
